docs/cni/kubernetes/AWS.md

```diff
@@ -176,7 +176,12 @@ aws ec2 run-instances \
 #  Include the subnet param above if using a non-default VPC
 ```
 
-Deploy at least one worker node using the following command:
+Now, deploy at least one worker node.
+
+First, edit `cloud-config/node-config.yaml` and replace all instances of `kubernetes-master` with your Master's private
+DNS name.  This can be found in the output of the previous command, or the AWS portal.
+
+Then, run the following command to start a node instance.
 
 ```
 aws ec2 run-instances \
```

Comment:
Hm, not sure how you get it for the default VPC.  Probably a command for it, or can get it through the portal.

I didn't mind the commenting here since in the case you need to uncomment it, you have the value, and in all other cases we don't use the value.  It does make it harder to copy and paste this command when using a non-default VPC.